src/Resources/views/Ckeditor/browser.html.twig

```diff
@@ -247,21 +257,23 @@ file that was distributed with this source code.
 {% endblock %}
 
 {% block list_filters %}
-    {% if admin.datagrid.filters %}
-        {% form_theme form admin.getTemplate('filter') %}
+    {% if datagrid.filters %}
+        {#        {% form_theme form admin.getTemplate('filter') %}#}
```

Comment:
?

Reply:
Neither the property "getTemplate" nor one of the methods "getTemplate()", "getgetTemplate()"/"isgetTemplate()"/"hasgetTemplate()" or "__call()" exist and have public access in class "Sonata\MediaBundle\Admin\ORM\MediaAdmin".